scripts/wrappers/version.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+
+import click
+import json
+from common.utils import snap_data
+from common.utils import run
+
+
+VERSIONS_FILE_NAME = "versions.json"
+VERSIONS_FILE_PATH = snap_data() / VERSIONS_FILE_NAME
+
+
+VERSIONS = None
+
+
+def get_upstream_version(upstream: str) -> str:
+    global VERSIONS
+
+    if VERSIONS is None:
+        # Cache versions
+        VERSIONS = _read_versions_file()
+
+    return VERSIONS[upstream]
+
+
+def _read_versions_file():
+    with open(VERSIONS_FILE_PATH, mode="r") as versions_file:
+        versions = json.loads(versions_file.read())
+        return versions
+
+
+def get_snap_version_data():
+    output = run("snap", "list", "microk8s")
+    snap_info = output.split("\n")[1]
+    version, revision = snap_info.split()[1:3]
+    return version, revision
+
+
+def print_versions() -> None:
+    version, revision = get_snap_version_data()
+    print(f"MicroK8s {version} revision: {revision}")
+
+    kube_version = get_upstream_version("kube")
```

Comment:
Isn't it better to just load the file once, then read from the dict? 
e.g.

```
versions = json.load(version_file)
kube_version = version["kube"]
cni_version = version["cni"]
```

Reply:
yup, indeed. much simpler 👍🏼